### PR TITLE
[Transform] disable bwc for backport epoch_millis change

### PR DESCRIPTION
**Pull request by hendrikmuhs**
disable bwc to integrate 65952

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -176,7 +176,7 @@ tasks.register("verifyVersions") {
  */
 
 boolean bwc_tests_enabled = false
-final String bwc_tests_disabled_issue = "https://github.com/elastic/elasticsearch/pull/65927" /* place a PR link here when committing bwc changes */
+final String bwc_tests_disabled_issue = "https://github.com/elastic/elasticsearch/pull/65952" /* place a PR link here when committing bwc changes */
```
Comment (przemekwitek) [inline build.gradle:179]: Could you remind me what happens after your change is integrated?
Will you revert this PR number to `65927`?

**Review by przemekwitek — APPROVED** [review]
LGTM

File: build.gradle (hunk 1 above)
Comment (hendrikmuhs) [reply build.gradle:179]: I am tackling that offline with @jimczi, normally this field is empty, but we both have BWC at the moment.

**Comment by hendrikmuhs** [comment]
closing as another BWC backport is in flux.

**Pull request closed without merging**